src/integration/java/com/smartcar/sdk/VehicleTest.java

```diff
@@ -137,6 +139,23 @@ public void testActionUnlock() throws SmartcarException {
         this.vehicle.unlock();
     }
 
+    /**
+     * Tests that the batch request method works.
+     */
+    @Test(groups = "vehicle")
+    public void testBatch() throws SmartcarException {
+        ArrayList<String> paths = new ArrayList<>();
```

Comment:
Normally, in Java you set the type of the variable to be the interface class.

```suggestion
        List<String> paths = new ArrayList<>();
```